imports/plugins/core/graphql/lib/hocs/withPrimaryShop.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Query } from "react-apollo";
-import getPrimaryShop from "../queries/getPrimaryShop";
+import getShop from "../queries/getShop";
```

Comment:
Yes my thinking was that things using `withPrimaryShop` should be switched at some point to use `withShop`. This is to enable adding back a shop switcher at some point soon. But there were a bunch of places so I didn't want to change them all in this PR. I will add a deprecation comment here, though.

Reply:
Sounds good.

Could we also just have `withPrimaryShop` call the `withShop` mutation, instead of having almost duplicate code? That way we can confirm that it actually is the same thing no matter which of the two are being used.

Reply:
Yeah I will try that

Reply:
It turned out there were not that many places using `withPrimaryShop` and I was able to replace them with `withPrimaryShopId` + `withShop` HOCs together. Eventually we can work to eliminate `withPrimaryShopId` and use a switchable reactive shopId instead.